py/torch_tensorrt/dynamo/conversion/impl/grid.py

```diff
@@ -7,10 +7,10 @@
 from torch_tensorrt.fx.converters.converter_utils import set_layer_name
 from torch_tensorrt.fx.types import TRTTensor
```

Comment:
Not related to this PR, but could you change this from `torch_tensorrt.fx` to `torch_tensorrt.dynamo` ?